src/main/java/uk/gov/companieshouse/api/testdata/service/impl/CompanyProfileServiceImpl.java

```diff
@@ -115,16 +120,15 @@ private Links createLinks(String companyNumber) {
         return links;
     }
 
-    private Address createRoa() {
-
-        Address registeredOfficeAddress = new Address();
-
-        registeredOfficeAddress.setAddressLine1("Crown Way");
-        registeredOfficeAddress.setCountry("United Kingdom");
-        registeredOfficeAddress.setLocality("Cardiff");
-        registeredOfficeAddress.setPostalCode("CF14 3UZ");
-
-        return registeredOfficeAddress;
+    private String getJurisdictionString(Jurisdiction jurisdiction) {
+        switch(jurisdiction) {
+            case ENGLAND_WALES:
+                return "england-wales";
+            case SCOTLAND:
+                return "scotland";
```

Comment:
Can we move this logic to the `Jurisdiction` enum by adding a string field to it and so that we can call the `toString` method instead? 
We probably can use that field to serialize/deserialize json rather than duplicating the json property values if they match.